cmd/ocm-backplane/login/login.go

```diff
@@ -119,47 +130,29 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 	logger.Debugf("Backplane Config File Contains: %v \n", bpConfig)
 
+	// login to the cluster based on login type
 	logger.Debugf("Extracting Backplane Cluster ID")
-	// Currently go-pagerduty pkg does not include incident id validation.
-	if args.pd != "" {
-		if bpConfig.PagerDutyAPIKey == "" {
-			return fmt.Errorf("please make sure the PD API Key is configured correctly in the config file")
-		}
-		pdClient, err := pagerduty.NewWithToken(bpConfig.PagerDutyAPIKey)
+	switch loginType {
+	case LoginTypePagerduty:
+		info, err := getClusterInfoFromPagerduty(bpConfig)
+		clusterKey = info.ClusterID
+		elevateReason = info.WebURL
 		if err != nil {
```

Comment:
small nit, maybe move the error handle right after the `getClusterInfoFromPagerduty()`

Reply:
done